python/pyarrow/array.pyx

```diff
@@ -217,11 +220,13 @@ def from_pylist(object list_obj, DataType type=None):
     return box_arrow_array(sp_array)
 
 
-def from_pandas_series(object series, object mask=None):
+def from_pandas_series(object series, object mask=None, timestamps_to_ms=False):
```

Comment:
This is a convenience option so you can easily write Parquet files with timestamp columns.